src/plugins/expression_shape/public/components/shape_component.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState, useEffect, lazy, useCallback, RefCallback } from 'react';
+import { useResizeObserver } from '@elastic/eui';
+import {
+  ShapeAttributes,
+  ShapeContentAttributes,
+  SvgConfig,
+  getDefaultShapeData,
+  withSuspense,
+  ShapeRef,
+} from '../../../presentation_util/public';
+import { Dimensions, ShapeComponentProps } from './types';
+import { getViewBox } from '../../common/lib';
+
+const LazyShapeDrawer = lazy(() => import('./shape_drawer'));
```

Comment:
Can you just import LazeShape drawer from ../ instead of having to do Lazy again here?

Reply:
Done.